### PR TITLE
Update retry-service-specific.md

### PR DESCRIPTION
**Pull request by brianharwell**
Fixed a few grammatical errors

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/best-practices/retry-service-specific.md](https://github.com/brianharwell/architecture-center/blob/patch-1/docs/best-practices/retry-service-specific.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/best-practices/retry-service-specific?branch=pr-en-us-38) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C10%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201704101530358937-38%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/best-practices/retry-service-specific.md [hunk 1]
```diff
@@ -684,7 +684,7 @@ The following table shows the default settings for the built-in retry policy.
 | ConfigurationOptions |ConnectRetry<br /><br />ConnectTimeout<br /><br />SyncTimeout |3<br /><br />Maximum 5000 ms plus SyncTimeout<br />1000 |The number of times to repeat connect attempts during the initial connection operation.<br />Timeout (ms) for connect operations. Not a delay between retry attempts.<br />Time (ms) to allow for synchronous operations. |
 
 > [!NOTE]
-> SyncTimeout contributes to the end-to-end latency of an operation. However, in general, using synchronous operations is not recommended. For more information see [Pipelines and Multiplexers](http://github.com/StackExchange/StackExchange.Redis/blob/master/Docs/PipelinesMultiplexers.md).
+> SyncTimeout contributes to the end-to-end latency of an operation. However, in general, using synchronous operations are not recommended. For more information see [Pipelines and Multiplexers](http://github.com/StackExchange/StackExchange.Redis/blob/master/Docs/PipelinesMultiplexers.md).
```
Comment (MikeWasson) [inline docs/best-practices/retry-service-specific.md]: Here "is" is correct ("using ... is")

**Comment by brianharwell** [comment]
Are you sure? Operations is plural so the correct usage is "are". "Is" would be appropriate if we were dealing with a collective noun but operations is not a collective noun.

https://www.grammarly.com/blog/is-vs-are/

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/best-practices/retry-service-specific.md](https://github.com/brianharwell/architecture-center/blob/patch-1/docs/best-practices/retry-service-specific.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/best-practices/retry-service-specific?branch=pr-en-us-38) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C17%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201704171728318504-38%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by MikeWasson** [comment]
@brianharwell The subject is the gerund 'using' - see https://english.stackexchange.com/a/18818 (Enslish grammar is weird...) - that said, I think the original sentence was awkward, so I've revised it. Thanks for the suggestions!

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/best-practices/retry-service-specific.md](https://github.com/brianharwell/architecture-center/blob/patch-1/docs/best-practices/retry-service-specific.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/best-practices/retry-service-specific?branch=pr-en-us-38) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C17%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201704171753409563-38%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**